synapse/lib/net.py

```diff
@@ -86,6 +89,9 @@ def _runPollLoop(self):
                     continue
                 logger.exception('plex thread error: %r' % (e,))
 
+            if not fems:
+                time.sleep(0.035)
```

Comment:
This value was chosen to be pretty small enough to cause a idle interpreter to show 0.0 - 0.1% cpu usage on osx.